app/helpers/elastic.js

```diff
@@ -5,23 +5,50 @@ var elasticsearch = require("elasticsearch");
 
 var variablesHelper = require("../helpers/variables");
 
-var variables = variablesHelper();
-var host = "";
-var log = "";
+var initiateClient = function initiateClient() {
+	var me = this;
 
-if (variables && variables.acpaassearch.variables.host) {
-	host = variables.acpaassearch.variables.host;
+	variablesHelper.reload()
+		.then(function(variables) {
+
+			var host = "";
+			var log = "";
+
+			if (!variables || !variables.acpaassearch.variables.host) {
+				me.connected = false;
+				return;
+			}
+
+			host = variables.acpaassearch.variables.host;
+
+			me.client = new elasticsearch.Client({
+				host: host,
+				log: log,
+			});
+
+			// Check if connection can be made
+			me.client.ping(function(err) {
+				if (err) {
+					console.log("Unable to initiate elastic client");
+					console.log(err);
```

Comment:
This can be removed I guess?
Also, are log's allowed according to the linter?

Reply:
These logs are intentional. 
The linter doesn't show an error because it's an old config `"@district01/eslint-config": "~1.0.0",`

Reply:
De linter werkt wel als ik de node versie 6.x gebruik. 🤷‍♂

Reply:
Deze PR bevat oude code die gewoon nog niet naar hvperdrive master gegaan is. De effectieve changes die ik uitgevoerd heb, ging ik via een andere PR doen. Ik ga op die PR de linting errors fixen.